src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputFormattedIncorrectJavadocParagraph.java

```diff
@@ -0,0 +1,118 @@
+package com.google.checkstyle.test.chapter7javadoc.rule712paragraphs;
+
+/**
+ * Some Javadoc.
+ *
+ * <p>/^ WARN/ Some Javadoc.
+ *
+ * <p>
+ */
+class InputFormattedIncorrectJavadocParagraph {
+  // violation 3 lines above '<p> tag should be placed immediately before the first word'
+
+  // violation 4 lines below '<p> tag should be placed immediately before the first word'
+  /**
+   * Some Javadoc.
+   *
+   * <p>
+   *
+   * <p>Some Javadoc.
+   *
+   * @since 8.0
+   */
+  public static final byte NUL = 0;
+
+  /**
+   * Some
+   *
+   * <p>Javadoc.
+   *
+   * <p>Some Javadoc.
+   *
+   * @see <a
+   *     href="http://www.gwtproject.org/doc/latest/DevGuideOrganizingProjects.html#DevGuideModules">
+   *     Documentation about GWT emulated source</a>
+   */
+  boolean emulated() {
+    return false;
+  }
+
+  // violation 6 lines below '<p> tag should be placed immediately before the first word'
+  // violation 7 lines below '<p> tag should be placed immediately before the first word'
+  // violation 8 lines below '<p> tag should be placed immediately before the first word'
+  /**
+   * Some Javadoc.
+   *
+   * <p>
+   *
+   * <p>
+   *
+   * <p>
+   *
+   * <p>/^WARN/ Some Javadoc.
+   *
+   * <p>
+   */
+  class InnerInputCorrectJavaDocParagraphCheck {
+    // violation 3 lines above '<p> tag should be placed immediately before the first word'
+
+    // violation 4 lines below '<p> tag should be placed immediately before the first word'
+    /**
+     * Some Javadoc.
+     *
+     * <p>
+     *
+     * @since 8.0
+     */
+    public static final byte NUL = 0;
+
+    // violation 8 lines below '<p> tag should be placed immediately before the first word'
+    /**
+     * /^WARN/ Some Javadoc.
+     *
+     * <p>/^WARN/
+     *
+     * <p>/^WARN/ Some Javadoc.
+     *
+     * <p>
+     *
+     * @see <a
+     *     href="http://www.gwtproject.org/doc/latest/DevGuideOrganizingProjects.html#DevGuideModules">
+     *     Documentation about GWT emulated source</a>
+     */
+    boolean emulated() {
+      return false;
+    }
+  }
+
+  InnerInputCorrectJavaDocParagraphCheck anon =
+      new InnerInputCorrectJavaDocParagraphCheck() {
+
+        /**
+         * Some Javadoc.
+         *
+         * <p>Some Javadoc.
+         *
+         * @since 8.0
+         */
+        public static final byte NUL = 0;
+
+        // violation 4 lines below '<p> tag should be placed immediately before the first word'
```

Comment:
Is it a bug in the formatter?

Reply:
Yes, I had the same question.
But formatter might not bother about this at all. 
@Zopsss , please confirm.

Reply:
[7.1.2 Paragraphs](https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs) says:

> Each paragraph except the first has <p> immediately before the first word, with no space after it.

so `<p>` should be placed immediately before `/WARN/ Some Javadoc.`, something like:

```
<p>/WARN/ Some Javadoc.
```

so this might be a formatter's bug, will open an issue regarding this on formatter's GH

Reply:
formatter do care a bit about P tags - https://github.com/google/google-java-format/issues?q=is%3Aissue+p+tag+is%3Aclosed

let close issue on them and let move forward with merge of this PR.

Reply:
it is grey area, there is no words, so there could be multiple of interpretations of

> Each paragraph except the first has `<p>` immediately before the first word, with no space after it.


https://github.com/google/google-java-format/issues/1150